src/schema/v2/partner.ts

```diff
@@ -214,15 +214,20 @@ export const PartnerType = new GraphQLObjectType<any, ResolverContext>({
                   artwork_id: artworkIds,
                 }
 
+                const totalCount = parseInt(headers["x-total-count"] || "0", 10)
+
                 return partnerArtworksAllLoader(id, gravityArtworkArgs).then(
                   ({ body }) => {
-                    return connectionFromArraySlice(body, args, {
-                      arrayLength: parseInt(
-                        headers["x-total-count"] || "0",
-                        10
-                      ),
-                      sliceStart: offset,
-                    })
+                    return {
+                      totalCount,
+                      ...connectionFromArraySlice(body, args, {
+                        arrayLength: parseInt(
```

Comment:
Looks like we're repeating the `totalCount` logic here. Could we make this `arrayLength: totalCount`?

Reply:
Oh yes good call!